spec/requests/api/v1/events/events_spec.rb

```diff
@@ -99,11 +95,8 @@
     event.reload
     expect(event.name).to be
     expect(response_json).to eq(
-      'message' => 'Validation Failed',
-      'errors' => [
-        "Name can't be blank"
-      ]
+      'errors' => "Validation failed: Name can't be blank"
```

Comment:
I rewrote the test to test for 
```
expect(event.name).to eq ('Original Name')
```
Keeping the 'Name Can't Be Blank,' since that's why the PATCH request fails. How can I test a PATCH request without using .reload?